jarviscli/plugins/routine.py

```diff
@@ -0,0 +1,30 @@
+from plugin import plugin
+
+@plugin("create routine")
+def new_routine(jarvis, s):
+    commands = []
+    while True:
+        jarvis.say("Type a command, or press enter to finish the routine")
+        command = jarvis.input()
+        if command == "":
+            break
+        commands.append(command)
+    jarvis.create_routine(commands, s)
```

Comment:
You should check 's' is not empty string. First time ran, I was quite irritated, because managed to created routine without a name ;).